package.json

```diff
@@ -160,6 +175,7 @@
         "react-textarea-autosize": "8.4.0",
         "remark-gfm": "3.0.1",
         "remark-parse": "10.0.1",
+        "scheduler": "0.23.0",
```

Comment:
And that scheduler? Do we need to execute a cron job?

Reply:
Same as previous comment